### PR TITLE
added function and test reverseStr

### PR DESCRIPTION
**Pull request by LanaG1011**
reverseStr function and reverseStr test added

### CODE REVIEW

**Review by TimothyHuxx — APPROVED** [review]
Approved

File: test/reverseStr_Func_SvetaG.spec.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const reverseStr = require("../src/reverseStr_Func_SvetaG");
+const expect = require("chai").expect;
+
+describe("Test suit for reverseStr function", () => {
+    it("reverseStr(Abc) should return (cbA)", () => {
+        expect(reverseStr("Abc")).to.be.equal("cbA");
+    })
+})
```
Comment (shabaiev) [inline test/reverseStr_Func_SvetaG.spec.js]: can you remove the new line at line 9?

**Review by LIliia149 — APPROVED** [review]
approved

**Review by shabaiev — COMMENTED** [review]
niiiiice

**Pull request merged**